.github/update-sync-param-files.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+from pathlib import Path
+
+import yaml
+
+
+def src_name_to_dst_name(src: Path) -> Path:
+    module_launch_pkg_name = src.parents[len(src.parents) - 3].stem  # e.g. "tier4_control_launch"
+    file_path_under_config = src.relative_to(Path("launch") / module_launch_pkg_name / "config")
+    dst = Path("autoware_launch/config") / module_launch_pkg_name / file_path_under_config
+
+    return dst
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("input_universe_dir", type=str, help="path to the target autoware.universe")
```

Comment:
As I wrote in https://github.com/autowarefoundation/autoware_launch/pull/75/files#r952089227, if you clone the repo in this file, I think you don't have to add this arg.